nature_of_code/chp_03_oscillation/3_01_angular_motion.rs

```diff
@@ -11,49 +12,52 @@ fn main() {
 
 struct Model {
     angle: f32,
-    a_velocity: f32,
-    a_acceleration: f32,
+    angle_velocity: f32,
+    angle_acceleration: f32,
 }
 
 fn model(app: &App) -> Model {
     let angle = 0.0;
-    let a_velocity = 0.0;
-    let a_acceleration = 0.0001;
+    let angle_velocity = 0.0;
+    let angle_acceleration = -0.0001;
     app.new_window().size(800, 200).view(view).build().unwrap();
     Model {
         angle,
-        a_velocity,
-        a_acceleration,
+        angle_velocity,
+        angle_acceleration,
     }
 }
 
 fn update(_app: &App, model: &mut Model, _update: Update) {
-    model.angle += model.a_velocity;
-    model.a_velocity += model.a_acceleration;
+    model.angle += model.angle_velocity;
+    model.angle_velocity += model.angle_acceleration;
 }
 
 fn view(app: &App, model: &Model, frame: Frame) {
     // Begin drawing
     let draw = app.draw();
-    draw.background().color(WHITE);
+    draw.rect().wh(app.window_rect().wh()).color(WHITE);
 
     draw.line()
         .start(pt2(-60.0, 0.0))
         .end(pt2(60.0, 0.0))
-        .color(BLACK)
```

Comment:
not really sure why this was taken out and replaced with `color(rgb8(0,0,0))`. Can we change it back to BLACK as this doesn't seem to add anything other than further complexity.